app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] In order to match the ERD diagram in our project requirements, let us kindly change the `author_id` in both the comment and like models to `user_id`. It's only the post model that should have the `author_id` as it's column name
![stu](https://github.com/Rishi-Mishra0704/blog-app/assets/46388113/7618bb51-c8ad-4702-9e22-85b01dd4fa6b)

Reply:
hi @addod19 if I change the column name will it affect my record and schema?

Reply:
Please reply so I can start working

Reply:
@addod19 I still haven't heard from you